css/portfolio.css

```diff
@@ -0,0 +1,205 @@
+* {
+	box-sizing: border-box;
+/*	border: 1px solid red;*/
+}
+
+@font-face {
+    font-family: Modenine;
+    src: url('../font/MODENINE.TTF');
+    src: url('../font/MODENINE.woff');
+}
+
+body {
+    font-family: Modenine;
+    margin: 0;
+}
+
+
+a {
+    color: #000000;
+    text-decoration: none;
+}
+
+.fullscreen-bg {
+    position: absolute;
+    top: 0;
+    right: 0;
+    left: 0;
+    overflow: hidden;
+    z-index: 2;
+    height: 100vh;
+
+}
+
+.fullscreen-bg__video {
+    position: absolute;
+    top: 0px;
+    left: 0;
+    min-height: 100vh;
+    min-width: 100%;
+    /*transform: translateX(-50%);*/
+    /*left: 50%;*/
+}
+
+.header{
+    font-size: small;
+    position: relative;
+    z-index: 10;
+    height: 100vh;
+    overflow: visible;
+    /*background-color:#333;*/
+}
+
+.header-title{
+    color: whitesmoke;
+    margin: 0;
+    position: absolute;
+    text-align: center;
+    top: 50%;
+    margin-top: -26px;
+    width: 100%;
+}
+
+.logo {
+    border-left-width: 10px;
+    height: 90px;
+    position: absolute;
+    top: 10px;
+    left:50%;
+    z-index: 10;
+}
+
+
+.vl {
```

Comment:
this div is not working on the bio section on mobile

Reply:
i just changed the margin a moved it more to the right